app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -486,11 +491,57 @@ class MainActivity : BaseActivity() {
         syncAutoConfigActivityResultLauncher.launch(Intent(this, SyncAutoConfigActivity::class.java))
     }
 
+    fun easterEggConfettiHasBeenClicked(from: String) {
+
+        val currentTime = System.currentTimeMillis()
+
+        if (easterEggConfettiTime == 0L || currentTime - easterEggConfettiTime > EASTER_EGG_CONFETTI_DELAY_TRIGGER) {
+            easterEggConfettiTime = currentTime
+            easterEggConfettiCount = 1
+        } else {
+            easterEggConfettiCount++
+        }
+
+        if (easterEggConfettiCount == EASTER_EGG_CONFETTI_TAPS_TRIGGER) {
+            mainViewModel.easterEggConfettiTrigger.value = from
```

Comment:
Create a function inside UiUtils that starts the confetti instead of a live data